InvenTree/plugin/base/supplier/models.py

```diff
@@ -0,0 +1,75 @@
+"""Model defintion for supplier mixin."""
+from django.db import models
+from django.utils.translation import gettext_lazy as _
+
+import common.models
+from plugin import registry
+from plugin.models import PluginConfig
+from plugin.plugin import InvenTreePlugin
+
+
+class ConnectionSetting(common.models.BaseInvenTreeSetting):
```

Comment:
Note: This class needs some rework after merging master in (Ref: #4834 )

Reply:
probably, a lot has changed in this area